src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -2002,3 +1984,49 @@ def _get_member_groups(get_member_group_func, identifier, security_enabled_only)
         "securityEnabledOnly": security_enabled_only
     }
     return get_member_group_func(identifier, body)
+
+
+class RoleApiHelper:
```

Comment:
Why not just rename the `MultiAPIAdaptor` in`_multi_api_adaptor.py` and updated related code? `custom.py` has already been a bit large